### PR TITLE
merge fiha with master

### PR DESCRIPTION
**Pull request by nn-fiha**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,209 @@
...
+
+   npm install mongoose
+   
+   npm run dev
+
+   cd ..
+
+   ```
+
+  **Insturctions for Backend**
```
Comment (Parvej-Hoque-Palash) [inline README.md]: Could you please change this line to "Instructions for Frontend"-

File: backend/controllers/adminController/out/adminDashboard.js.html [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+
+<div id="main">
+
+    <h1 class="page-title">Source: adminDashboard.js</h1>
+
+    
+
+
+
+
```
Comment (Parvej-Hoque-Palash) [inline backend/controllers/adminController/out/adminDashboard.js.html:26]: Could you please remove extra spaces for beautification.

File: backend/controllers/adminController/out/index.html [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+
+    <h1 class="page-title">Home</h1>
+
+    
+
+
+
+    
+
+
```
Comment (Parvej-Hoque-Palash) [inline backend/controllers/adminController/out/index.html:28]: Could you please remove those extra spaces.

File: backend/controllers/adminController/out/index.html [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+
+
+
+
+
+
+
```
Comment (Parvej-Hoque-Palash) [inline backend/controllers/adminController/out/index.html:49]: Could you please remove those extra spaces.

**Review by Parvej-Hoque-Palash — APPROVED** [review]

**Review by Tazim381 — APPROVED** [review]

**Pull request merged**